pkg/roachprod/install/cockroach.go

```diff
@@ -140,7 +140,8 @@ type StartOpts struct {
 
 	// IsRestart allows skipping steps that are used during initial start like
 	// initialization and sequential node starts.
-	IsRestart bool
+	IsRestart              bool
+	OverwriteStartupScript bool
```

Comment:
Nit: this field could have some comments about when someone would want (or not want) to overwrite the startup script.

Reply:
As discussed I got rid of the flag since not overwriting in case of `restart=true` simplifies the logic